### DISCUSSION TITLE
Floating Label like the MUI select floating label

### DISCUSSION THREAD

**Discussion by OscarJVD** [JedWatson/react-select#5671] [Q&A]
How can I make a select with a floating label like the MUI select with floating label? I've tried but I'd like to know your solutions
![image](https://github.com/JedWatson/react-select/assets/57403552/b2b6231d-26d6-48f0-a28f-b299ce604fba)

**Comment by ebonow** [comment]
Hi Oscar, 

I had some working example for v3 but hadn't worked out the kinks yet for the changes in the css in the major version update. I'll see if I can get something together in the next day or so.

**Reply by OscarJVD** [reply]
I'd appreciate it a lot
